### DISCUSSION TITLE
Saber Dojo

### DISCUSSION THREAD

**Discussion by yoeyyutch** [ToniMacaroni/SaberFactory#112] [Ideas]
I think it would be useful to have some stationary cubes to swing at in the SF environment, possibly the entire grid or just the ones the player chooses.  Additional features might include live swing/accuracy stats, note debris, or a visualization of the cut plane through the note. or calling for specific notes to jump in as they do in the game. I'm getting kind of carried away but something that just came to mind was tossing a note up in the air and slicing it in half on the way down. It sounds so satisfying for some reason.

In reality, the full set of features that I'm suggesting could be its own training facility/dojo mod. I just found that in the SF environment, I liked having the saber in my hand without notes flying at me, but I still feel like I need something to swing at! So in my mind it feels like a natural progression of SF whether it is part of this mod or a launching point for a new one. 

Thanks for taking the time to read this. I appreciate the work you do!
Yoey Yutch

**Comment by ToniMacaroni** [comment]
Those are a lot of suggestions, but I will take a look at them.
Though it will probably be it's own mod (as an addon for sf).

**Reply by yoeyyutch** [reply]
I agree. Those are a lot of suggestions. I hope it goes without saying that I have no expectations for you to implement any of them. I'd be thrilled if you did, but it's all gravy. It's a cool mod as it is.  I'm not much of a coder but I could do some bug shooting, maybe graphics. Happy to help where I can. Let me know. Thanks.
